src/statsmodel.jl

```diff
@@ -193,6 +193,22 @@ function StatsBase.coeftable(model::TableModels; kwargs...)
     ct
 end
 
+_show_fit_stats(io::IO, model::TableModels) = nothing
+
+function _show_fit_stats(io::IO, model::TableRegressionModel)
+    try
+        println("R²: ", round(r2(model), sigdigits=4),
+             "\t Adjusted R²: ", round(adjr2(model), sigdigits=4))
+
+        fstat = fstatistic(model)
+        println(io, fstat)
```

Comment:
`fstatistic` doesn't exist anywhere currently, and only GLM provides the needed functionality via `ftest` (https://github.com/JuliaStats/GLM.jl/pull/424/). An empty `ftest` definition should be added to StatsAPI first. Probably the API can just require that the returned object has the `dof`, `fstat` and `pval` properties so that you can extract them and print them.
```suggestion
        ftr = ftest(model)
        @printf(io, "F-statistic: %.4f on %s degrees of freedom, p-value %s",
                ftr.fstat, ftr.dof, PValue(ftr.pvalue))
```

Reply:
@nalimilan what if adding a print API to StatsAPI so packages can implement their own printing?

Reply:
@xgdgsc every package is free to implement `Base.show` methods for their own types